lxd-agent/daemon.go

```diff
@@ -8,13 +8,17 @@ import (
 type Daemon struct {
 	// Event servers
 	events *events.Server
+
+	// The channel which is used to indicate that the lxd-agent was able to connect to LXD.
+	chConnected chan any
```

Comment:
Using an empty `struct{}` as type is clearer that the channel itself doesn't transfer any data, and is just used for signalling a single event has occurred.